code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -411,20 +411,28 @@ Congratulations! You are now trained for xenobiology research!"}
 	return 1
 
 /obj/item/weapon/abductor_baton/attack(mob/target, mob/living/user)
-	if(!IsAbductor(user))
+	if (!IsAbductor(user))
 		return
 
-	if(isrobot(target))
+	if (isrobot(target))
 		..()
 		return
 
-	if(!isliving(target))
+	if (!isliving(target))
 		return
 
 	var/mob/living/L = target
 
 	user.do_attack_animation(L)
-	switch(mode)
+
+	// [tgstation/#14599] Allow reactive teleport armor to kick in when attacked.
```

Comment:
That's a bad habit!

I'm all for removing that comment _if_ the original author of the `check_shields` proc shows up and tells me that this is a good fix. Otherwise I can't guarantee that this fix hasn't introduced a new bug -- and in that case you'll want to know why someone put that code there.